### DISCUSSION TITLE
Importing File to Message-Ix

### DISCUSSION THREAD

**Discussion by Anurag-Dey** [iiasa/message_ix#471] [Getting started] [answered]
Dear All,

I am new to message-ix. I am getting a Not Implemented Error while reading an excel file. I am attaching the snaps. Kindly look into the codes and let me know if I am doing this wrong and what is the solution to this.

![image (1)](https://user-images.githubusercontent.com/66878460/118771798-96403300-b8a0-11eb-8133-7bd6f02fa7f1.png)
![image (2)](https://user-images.githubusercontent.com/66878460/118771793-950f0600-b8a0-11eb-8ab9-259d4e6daaca.png)

Thanks & Regards,
Anurag

**Comment by francescolovat** [comment]
Hi @Anurag94-cool,

I see that there is a typo on the name of the excel file when using the `.read_excel()` method.

The excel files format should be `.xlsx`, **and not** `.xlxs`.

I hope that helps.

**Reply by Anurag-Dey** [reply]
Hey @francescolovat,

I have made the change and now this error is coming.  

![Issue 1](https://user-images.githubusercontent.com/66878460/118778993-3ea5c580-b8a8-11eb-92b9-7aa0fcce014a.PNG)
![Issue 2](https://user-images.githubusercontent.com/66878460/118778988-3e0d2f00-b8a8-11eb-8f59-51cbd6736ebb.PNG)
![Issue 3](https://user-images.githubusercontent.com/66878460/118778979-3cdc0200-b8a8-11eb-8c3c-cbbdf75af22e.PNG)

Thanks

**Comment by francescolovat** [comment, marked as answer]
@Anurag94-cool please make sure that you have that excel file in the right path with the right name.

Also, I'd avoid using spaces for naming files which paths need to be called, that's confusing for Windows (use e.g. `message_ix.xlsx`).

Could you also copy-paste (**avoiding screenshots**) the output of running `message-ix show-versions` in an _Anaconda Prompt_, so we know which version of the model you are using (see [this link](https://docs.messageix.org/en/master/install.html?highlight=show-versions#check-that-installation-was-successful) as reference).

Thanks.

**Reply by Anurag-Dey** [reply]
@francescolovat Your suggestion works. I have changed the file name to a name without space and this works. Thanks a lot.
